docs/source/virtual_ecosystem/implementation/abiotic_implementation.md

```diff
@@ -113,109 +226,203 @@ Under steady-state, the balance equation for the leaves in each canopy layer is
     - \lambda g_{v} \frac {e_{L} - e_{A}}{p_{A}} \\
     & = 0
 ```
 
-where $R_{abs}$ is absorbed radiation, $R_{em}$ emitted radiation, $H$
-the sensible heat flux, $\lambda E$ the latent heat flux, $\epsilon_{s}$ the
-emissivity of the leaf, $\sigma$ the Stefan-Boltzmann constant, $T_{L}$ the
-absolute temperature of the leaf, $T_{A}$ the absolute temperature of the air
-surrounding the leaf, $\lambda$ the latent heat of vapourisation of water,
-$e_{L}$ the effective vapour pressure of the leaf, $e_{A}$ the vapour
-pressure of air and $p_{A}$ atmospheric pressure. $g_{Ha}$ is the heat
-conductance between leaf and atmosphere, $g_{v}$ represents the conductance
-for vapour loss from the leaves as a function of the stomatal conductance $g_{c}$.
+where:
+
+$R_{abs}$:
+Shortwave radiation absorbed by the canopy ($\mathrm{W\,m^{-2}}$)
+
+$R_{em}$:
+Emitted longwave radiation from the canopy ($\mathrm{W\,m^{-2}}$)
+
+$H$:
+Sensible heat flux from the canopy to the air ($\mathrm{W\,m^{-2}}$)
+
+$Q_{LE}$:
+Latent heat flux associated with transpiration from the canopy to the air
+($\mathrm{W\,m^{-2}}$)
+
+$\epsilon_{s}$:
+Emissivity of the leaf (-), typically close to 1
+
+$\sigma$:
+Stefan–Boltzmann constant ($5.67 \times 10^{-8}\,\mathrm{W\,m^{-2}\,K^{-4}}$)
+
+$T_{L}$:
+Temperature of the leaf (°C)
+
+$T_{A}$:
+Temperature of the air surrounding the leaf (°C)
+
+$\lambda$:
+Latent heat of vapourisation of water ($\mathrm{kJ\,kg^{-1}}$)
+
+$e_{L}$:
+Effective vapour pressure of the leaf (kPa)
+
+$e_{A}$:
+Vapour pressure of air (kPa)
+
+$p_{A}$:
+Atmospheric pressure (kPa)
+
+$g_{v}$:
+Conductance for vapour loss from the leaves ($\mathrm{mol\,m^{-2}\,s^{-1}}$) as a
+function of the stomatal conductance $g_{c}$ ($\mathrm{s\,m^{-1}}$)
+
+### Air and canopy temperature update
 
 A challenge in solving this equation is the dependency of latent heat and emitted
-radiation on leaf temperature. We use a linearisation approach to solve the equation for
-leaf temperature and air temperature simultaneously after
-{cite:t}`maclean_microclimc_2021`.
+radiation on leaf temperature. This method estimates updated canopy and air temperatures
+by linearizing the canopy energy balance and applying a Newton iteration. This
+approach accounts for the strong temperature dependence of radiative losses and ensures
+numerical stability in canopy energy balance closure, following the method described by
+{cite:t}`yang_scope_2021`. The goal is to find the leaf temperature that closes the
+energy balance at the leaf surface, see previous section.
 
-The air temperature surrounding the leaf $T_{A}$ is assumed to be influenced
-by leaf temperature $T_{L}$, soil temperature $T_{0}$, and reference air
-temperature $T_{R}$ as follows:
+#### Newton Linearization
 
-$$g_{tR} c_{p} (T_{R} - T_{A}) + g_{t0} c_{p} (T_{0} - T_{A}) + g_{L} c_{p} (T_{L} - T_{A})
-= 0$$
+To iteratively solve for the leaf temperature that satisfies the energy balance $EB$, we
```

Comment:
Would generally try and avoid multiple letter variable names as they quickly become hard to interpret once you use them within more complex mathematical notation, think it would be better to call this $E_B$ or $B$ or something similar

Reply:
I was following here the paper that I cite, I understand your point